composer.json

```diff
@@ -16,6 +16,7 @@
     "spryker/kernel": "^3.0.0",
     "spryker/oms": "^6.0.0 || ^7.0.0 || ^8.0.0",
     "spryker/propel-orm": "^1.0.0",
+    "spryker/step-engine": "^3.0.0",
```

Comment:
But it's not used in the code. Why you need it here?

Reply:
I see a few

    use Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface;

etc

Reply:
as far as I remember, without this `interface` the _payment_ step will render a quite broken layout for the form…

Reply:
:+1: